babel.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function (api) {
+  const browserslistEnv = !api.env('test')
```

Comment:
For tests we want to skip the default **Browserlist** rules

We're taking ESM files, making as few modifications as possible for Node.js 16, then running them in Jest as CommonJS out the other end

We don't want Jest running browser transformations on our test assertions